module/documents/actor.mjs

```diff
@@ -624,14 +604,13 @@ export default class ActorEd extends Actor {
       }
 
       return changes.concat(
-
         e.changes.map( ( c ) => {
           // eslint-disable-next-line no-param-reassign
-          c = foundry.utils.duplicate( c );
+          c = futils.duplicate( c );
```

Comment:
@ChristopherSD  what doest _applyBaseEffects do?

Reply:
No idea. I thought you copied in here from the last system? XD 
This function was already there...

Reply:
:-P I'll have a look